README.md

```diff
@@ -60,3 +60,29 @@ your favourite editor.
 In the front matter (the stuff inside `---` in the beginning of the
 file) you can specify the internal order with `weight: 2` to get it in
 second place in the table of contents.
+
+## License
+
+Unless otherwise noted, the content of the Dev Handbook are licensed
+under CC-BY-SA-4.0.
+
+The code examples of the Dev Handbook are licensed under the terms and
```

Comment:
I know we use GPLv2-only in real source, but is it a good idea to have example code under such a restrictive license? This means anyone copying code from the Dev Handbook will immediately be forced to share their source code, right?

Reply:
This seems to be the recommended way for documentation, but if we don't think the example code here ever will be substantial we might as well remove GPLv2 and keep everything CC. That is fine for me.